theme/custom.scss

```diff
@@ -490,3 +490,10 @@ a {
   white-space: nowrap;
   overflow: hidden;
 }
+
+.sticky {
```

Comment:
Is this class needed? There are a variety of USWDS utility classes that you could use here, like `.position-sticky`.